.github/workflows/test_branches.yml

```diff
@@ -95,13 +95,6 @@ jobs:
           PYENV: pip
           PACKAGES: cython
 
-        - os: ubuntu-18.04
```

Comment:
Why did you want to remove `pyutilib` tests on branches?

Reply:
Trying to keep the number of tests a user gets hit with on branches down to a manager number, and on a list of tests, this one doesn't really provide a lot of actionable information (now that 6.0 is out).